src/resources/batches/BatchForm.tsx

```diff
@@ -41,6 +41,44 @@ const schema = yup.object({
     )
 })
 
```

Comment:
Hello @TahaKhanAbdalli - can you tell me why has this text block been moved around in file, please?

Reply:
This component was supposed to be used in `item form` for only showing `active vault location`. One approach was to repeat this whole block but I moved it around to `export` it so that it could be used in the item form.